src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(calculateStatistics(fromFileName), toFileName);
+
+    }
+
+    public int [] calculateStatistics(String fromFileName) {
+        File file = new File(fromFileName);
+        int [] result = new int[3];
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String splitInput = reader.readLine();
+
+            while (splitInput != null) {
+                String [] temp = new String[2];
+                temp = splitInput.split(",");
+                splitInput = reader.readLine();
+
+                switch (temp[0]) {
+                    case "supply":
+                        result[0] += Integer.parseInt(temp[1]);
+                        break;
+                    default:
+                        result[1] += Integer.parseInt(temp[1]);
+                        break;
+                }
```

Comment:
The default case in the switch statement is misleading because it assumes any non-'supply' input is a 'buy'. This could lead to incorrect behavior if the input file contains unexpected data. Consider validating the input or handling unexpected data explicitly.